src/helpers/AllBonds.ts

```diff
@@ -1,18 +1,18 @@
 import { StableBond, LPBond, NetworkID, CustomBond } from "src/lib/Bond";
 import { addresses } from "src/constants";
 
-import { ReactComponent as DaiImg } from "src/assets/tokens/DAI.svg";
-import { ReactComponent as OhmDaiImg } from "src/assets/tokens/OHM-DAI.svg";
-import { ReactComponent as FraxImg } from "src/assets/tokens/FRAX.svg";
-import { ReactComponent as OhmFraxImg } from "src/assets/tokens/OHM-FRAX.svg";
-import { ReactComponent as wETHImg } from "src/assets/tokens/wETH.svg";
+import DaiImg from "src/assets/tokens/DAI.svg";
+import OhmDaiImg from "src/assets/tokens/OHM-DAI.svg";
+import FraxImg from "src/assets/tokens/FRAX.svg";
+import OhmLusdImg from "src/assets/tokens/OHM-LUSD.svg";
+import wETHImg from "src/assets/tokens/wETH.svg";
+import LusdImg from "src/assets/tokens/LUSD.svg";
```

Comment:
I think broke something.

Reply:
fixed svg in https://github.com/OlympusDAO/olympus-frontend/pull/453/commits/8f44220d33b840e46e19d8d238c0eccd2f1e0dee